lib/rubocop/cop/rspec/output.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # NOTE: This is the same as the `Rails/Output` cop with minor changes.
+    module RSpec
+      # Checks for the use of output calls like puts and print in specs.
+      #
+      # @example
+      #   # bad
+      #   puts 'A debug message'
+      #   pp 'A debug message'
+      #   print 'A debug message'
+      class Output < Base
+        include RangeHelp
+
+        MSG = 'Do not write to stdout in specs.'
+        RESTRICT_ON_SEND = %i[ap p pp pretty_print print puts binwrite syswrite
+                              write write_nonblock].freeze
+
+        # @!method output?(node)
+        def_node_matcher :output?, <<~PATTERN
+          (send nil? {:ap :p :pp :pretty_print :print :puts} ...)
+        PATTERN
+
+        # @!method io_output?(node)
+        def_node_matcher :io_output?, <<~PATTERN
+          (send
+            {
+              (gvar #match_gvar?)
+              (const {nil? cbase} {:STDOUT :STDERR})
+            }
+            {:binwrite :syswrite :write :write_nonblock}
+            ...)
+        PATTERN
+
+        # rubocop:disable Metrics/CyclomaticComplexity
+        def on_send(node)
+          return if node.parent&.call_type? || node.block_node
+          return if !output?(node) && !io_output?(node)
+          return if node.arguments.any? { |arg| arg.type?(:hash, :block_pass) }
+
+          range = offense_range(node)
+
+          add_offense(range)
+        end
+        # rubocop:enable Metrics/CyclomaticComplexity
```

Comment:
Could you please add the directive to the specific line where the offense occurs, instead of using a range?
```suggestion
        def on_send(node) # rubocop:disable Metrics/CyclomaticComplexity
          return if node.parent&.call_type? || node.block_node
          return if !output?(node) && !io_output?(node)
          return if node.arguments.any? { |arg| arg.type?(:hash, :block_pass) }

          range = offense_range(node)

          add_offense(range)
        end
```